package.json

```diff
@@ -23,6 +24,24 @@
     "gulp-sourcemaps": "^1.6.0",
     "gulp-uglify": "^1.5.3",
     "leaflet-sprite": "^1.0.0",
-    "serve-static": "^1.14.1"
+    "serve-static": "^1.14.1",
+    "@bower_components/Chart.js": "nnnick/Chart.js#^2.1.5",
+    "@bower_components/angular": "angular/bower-angular#1.7.8",
+    "@bower_components/angular-animate": "angular/bower-angular-animate#1.7.8",
+    "@bower_components/angular-bootstrap": "angular-ui/bootstrap-bower#^1.3.3",
+    "@bower_components/angular-resource": "angular/bower-angular-resource#1.7.8",
+    "@bower_components/angular-route": "angular/bower-angular-route#1.7.8",
+    "@bower_components/angular-sanitize": "angular/bower-angular-sanitize#1.7.8",
+    "@bower_components/bootstrap": "twbs/bootstrap#^3.3.6",
+    "@bower_components/font-awesome": "FortAwesome/Font-Awesome#^4.6.3",
+    "@bower_components/html5shiv": "aFarkas/html5shiv#^3.7.3",
+    "@bower_components/jquery": "jquery/jquery-dist#1.9.1 - 3",
+    "@bower_components/leaflet": "vperron/leaflet-dist#^0.7.2",
+    "@bower_components/moment": "moment/moment#^2.13.0",
+    "@bower_components/respond": "scottjehl/Respond#^1.4.2",
+    "@bower_components/underscore": "jashkenas/underscore#^1.8.3"
```

Comment:
would these break by moving from bower to yarn?

https://github.com/pfarrar1/angular_1_boilerplate_project/blob/de44be5ea9de4a3f331362fc77b7ba864fc8f054/gulpfile.js#L42

https://github.com/pfarrar1/angular_1_boilerplate_project/blob/de44be5ea9de4a3f331362fc77b7ba864fc8f054/gulpfile.js#L79

https://github.com/pfarrar1/angular_1_boilerplate_project/blob/de44be5ea9de4a3f331362fc77b7ba864fc8f054/gulpfile.js#L102

https://github.com/pfarrar1/angular_1_boilerplate_project/blob/de44be5ea9de4a3f331362fc77b7ba864fc8f054/gulpfile.js#L133

are there other references to include bower_components in other parts of the build pipeline?